chain/sync.go

```diff
@@ -674,6 +674,10 @@ func (syncer *Syncer) ValidateBlock(ctx context.Context, b *types.FullBlock) (er
 	//nulls := h.Height - (baseTs.Height() + 1)
 
 	// fast checks first
+	nulls := h.Height - (baseTs.Height() + 1)
+	if tgtTs := baseTs.MinTimestamp() + build.BlockDelaySecs*uint64(nulls+1); h.Timestamp != tgtTs {
```

Comment:
Why is it asked `h.Timestamp == tgtTs`? 
According to the spec, should it be `h.Timestamp >= tgtTs`?

Reply:
We will update the spec, allowing higher timestamps is a soft security issue.